README.wget-at.md

```diff
@@ -1,26 +1,35 @@
 {{REPO_NAME}}
 =============
 
-More information about the archiving project can be found on the ArchiveTeam wiki: [{{PROJECT_NAME}}](http://archiveteam.org/index.php?title={{PROJECT_NAME}})
+More information about the archiving project can be found on the ArchiveTeam wiki: [{{PROJECT_NAME}}](https://wiki.archiveteam.org/index.php?title={{PROJECT_NAME}})
 
 Setup instructions
 =========================
 
-Be sure to replace `YOURNICKHERE` with the nickname that you want to be shown as, on the tracker. You don't need to register it, just pick a nickname you like.
+### General instructions
 
-In most of the below cases, there will be a web interface running at http://localhost:8001/. If you don't know or care what this is, you can just ignore it—otherwise, it gives you a fancy view of what's going on.
+Data integrity is very important in Archive Team projects. Please note the following important rules:
 
-**If anything goes wrong while running the commands below, please scroll down to the bottom of this page. There's troubleshooting information there.**
+* [Do not use proxies or VPNs](https://wiki.archiveteam.org/index.php/ArchiveTeam_Warrior#Can_I_use_whatever_internet_access_for_the_Warrior?).
+* Run the project using the either the Warrior or the project-specific Docker container as listed below. [Do not modify project code](https://wiki.archiveteam.org/index.php/ArchiveTeam_Warrior#I'd_like_to_help_write_code_or_I_want_to_tweak_the_scripts_to_run_to_my_liking._Where_can_I_find_more_info?_Where_is_the_source_code_and_repository?). Compiling the project dependencies yourself is no longer supported.
+* Only use your tracker nickname(s) on machines you personally operate. Nickname sharing makes it harder to inspect data if a problem arises.
```

Comment:
I'd clarify that it's fine to share nicknames between machines

Reply:
I tried to indicate that already by using the word "machines" but perhaps that was unclear. I revised it a bit, though it does make the sentence a bit longer. I did want to try to keep these points somewhat brief to increase the chance that they would be read.